src/main/java/com/redislabs/redistimeseries/RedisTimeSeries.java

```diff
@@ -200,6 +201,22 @@ public boolean create(
     return args;
   }
 
+  /**
+   * TS.DEL key fromTimestamp toTimestamp
+   *
+   * @param key
+   * @param from timestamp from
+   * @param to timestamp to
+   * @return
+   */
+  public boolean del(String key, long from, long to) {
+    try (Jedis conn = getConnection()) {
+      return sendCommand(conn, Command.DEL, key, Long.toString(from), Long.toString(to))
```

Comment:
@sazzad16 can't we avoid converting to String and then back to `byte[]`

Reply:
@gkorland We can. But that won't save/optimize anything. Because in Jedis
```
  public static final byte[] toByteArray(final long value) {
    return SafeEncoder.encode(String.valueOf(value));
  }
```
and in JDK
```
    public static String valueOf(long l) {
        return Long.toString(l);
    }
```